src/plugins/orangehrmClaimPlugin/Dao/ClaimDao.php

```diff
@@ -296,14 +293,25 @@ public function getClaimExpenseById(int $id): ?ClaimExpense
     public function getClaimRequestExpense(int $requestId, int $expenseId): ?ClaimExpense
     {
         $claimRequest = $this->getClaimRequestById($requestId);
-        return $this->getRepository(ClaimExpense::class)->findOneBy(['id' => $expenseId, 'claimRequest' => $claimRequest, 'isDeleted' => false]);
+        return $this->getRepository(ClaimExpense::class)->findOneBy(
+            ['id' => $expenseId, 'claimRequest' => $claimRequest, 'isDeleted' => false]
+        );
+    }
+
+    /**
+     * @param int $id
+     * @return ClaimRequest|null
+     */
+    public function getClaimRequestById(int $id): ?ClaimRequest
+    {
+        return $this->getRepository(ClaimRequest::class)->findOneBy(['id' => $id, 'isDeleted' => false]);
     }
 
     /**
      * @param int[] $ids
      * @return int
      */
-    public function deleteClaimExpense(ClaimRequest  $claimRequest, array $ids): int
+    public function deleteClaimExpense(ClaimRequest $claimRequest, array $ids): int
```

Comment:
```suggestion
    public function deleteClaimExpense(int $claimRequestId, array $ids): int
```
shall we go like this :thinking: 
Also update function's doc comment